### DISCUSSION TITLE
ScriptReference/Renderer-sortingOrder

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#641] [Notes]
# ScriptReference/Renderer-sortingOrder

https://docs.unity3d.com/ScriptReference/Renderer-sortingOrder.html

**Comment by radiatoryang** [comment]
For more granular per-object sorting control, make sure you look at the SortingGroup component. https://docs.unity3d.com/Manual/class-SortingGroup.html

**Comment by xeetsh** [comment]
If you are searching for a way to update the inspector value "Order in Layer" you are at the right place.
SpriteRenderer.sortingOrder indeed changes that value.
